src/transforms/dodge.js

```diff
@@ -45,7 +45,7 @@ function dodge(y, x, anchor, padding, options) {
     if (!X) throw new Error(`missing channel: ${x}`);
     X = coerceNumbers(valueof(X.value, X.scale !== undefined ? scales[X.scale] : identity));
```

Comment:
We should do the same with X.scale here. I’ll add to your branch.

Reply:
yes there was no issue with x in my tests (but it's nicer this way)